lib/openra.rb

```diff
@@ -21,7 +21,7 @@
   "twitter" => { name: "Twitter", url: "https://twitter.com/openRA" },
   "reddit" => { name: "Reddit", url: "https://www.reddit.com/r/openra" },
   "youtube" => { name: "Youtube", url: "https://www.youtube.com/channel/UCRoiPL1J4K1-EhQeNazrYig"},
-  "gplus" => { name: "Google+", url: "https://www.google.com/+OpenraNet" },
+  "discord" => { name: "Discord", url: "https://discord.gg/UcUztfr" },
```

Comment:
This breaks the colour-ordering of the icons.  IMO the best ordering now is to put discord between twitter and reddit, and then move moddb to the immediate left of steam.